src/Kafka/Metrics/components/MetricsLagAlert.tsx

```diff
@@ -0,0 +1,32 @@
+import { VoidFunctionComponent } from "react";
+import { Alert, AlertActionCloseButton, Card } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export type MetricsLagAlertProps = {
+  isClosed: boolean;
+  onClickClose: () => void;
+};
+
+export const MetricsLagAlert: VoidFunctionComponent<MetricsLagAlertProps> = ({
+  isClosed,
+  onClickClose,
+}) => {
+  const { t } = useTranslation("metrics");
+
+  if (!isClosed) {
+    return (
+      <Card>
+        <Alert
+          isInline
+          variant="info"
+          title={t("metrics_lag_title")}
+          actionClose={<AlertActionCloseButton onClose={onClickClose} />}
```

Comment:
I see the Close action is included, but should there be any behavior for this represented in storybook, or is that handled when we bring these updates into the kas-ui or kafka-ui repos?

Reply:
you can refer to https://redhat-developer.github.io/app-services-ui-components/?path=/story/kafka-metrics-components-metrics-lag-alert--lag-message story @jgiardino, if you click on close button in the alert, the alert will be removed

Reply:
Got it. Thanks for clarifying!